src/pymodaq_plugins_signal_recovery/daq_move_plugins/daq_move_Lockin_DSP7265.py

```diff
@@ -0,0 +1,273 @@
+            if self.settings.child('imode').value() == DSP7265ThreadSafe.IMODES[2]:
+                self.controller.sensitivity = build_dict_from_float_list(
+                    [s * DSP7265ThreadSafe.SEN_MULTIPLIER[2]
+                     for s in DSP7265ThreadSafe.SENSITIVITIES],
+                    "A"
+                )[self.settings.child('sensitivity').value()]
+        elif param.name() == "voltage":
+            self.controller.voltage = param.value()
+        elif param.name() == "gain":
+            self.controller.gain = param.value()
+        else:
+            pass
+
+    def ini_stage(self, controller: object = None) -> edict:
+        """Actuator communication initialization
+
+        Parameters
+        ----------
+        controller: (object)
+            custom object of a PyMoDAQ plugin (Slave case). None if only one
+            actuator by controller (Master case)
+
+        Returns
+        -------
+        info: str
+            Id of the lockin
+        initialized: bool
+            False if initialization failed otherwise True
+        controller: object
+            Controller of the daq_move
+        """
+        try:
+            self.status.update(
+                edict(info="", controller=None, initialized=False)
+            )
+            if (self.settings.child('multiaxes', 'ismultiaxes').value()
+                    and self.settings.child('multiaxes',
+                                            'multi_status').value() == "Slave"):
+                if controller is None:
+                    raise Exception('No controller has been defined externally'
+                                    'while this axe is a slave one')
+                else:
+                    self.controller = controller
+            else:
+                adapter = ADAPTERS[self.settings.child('adapter').value()](
+                    self.settings.child('address').value()
+                )
+                self.controller = DSP7265ThreadSafe(adapter)
+
+            self.status.info = self.controller.id
+            self.settings.child('info').setValue(self.status.info)
+            self.status.controller = self.controller
+            self.status.initialized = True
+            return self.status
+
+        except Exception as e:
+            self.emit_status(
+                ThreadCommand('Update_Status', [getLineInfo() + str(e), 'log'])
+            )
+            self.status.info = getLineInfo() + str(e)
+            self.status.initialized = False
+            return self.status
+
+    def move_abs(self, f: DataActuator) -> None:
+        """ Move the actuator to the absolute target defined by value
+
+        Parameters
+        ----------
+        value: (float) value of the absolute target frequency
+        """
+        f = self.check_bound(f)
+        f = self.set_position_with_scaling(f)
+        self.controller.frequency = f.value()
+
+        self.target_frequency = f
```

Comment:
do not change current_value to current_frequency, same for target as those are used elsewhere in the parent class!!!!